CODING_STANDARD

```diff
@@ -92,6 +92,34 @@ Header files:
   of the header file rather than in line
 - Guard headers with #ifndef CPROVER_DIRECTORIES_FILE_H, etc
 
+Make files
+- Each source file should appear on a separate line
+- The final source file should still be followed by a trailing slash
+- The last line should be a comment to not be deleted, i.e. should look like:
+```
+SRC = source_file.cpp \
+      source_file2.cpp \
+      # Empty last line
+```
+- This ensures the Makefiles can be easily merged.
```

Comment:
Do you think it is worth adding the rule now, given that everything is out of order? If so how do you want to tackle it? I suppose it wouldn't be too hard to write a script to apply these changes to all the Makefiles and check in en masse.